test/multiapi/Expected/AcceptanceTests/Multiapi/multiapi/_operations_mixin.py

```diff
@@ -100,15 +100,15 @@ def test_one(
         message=None,  # type: Optional[str]
         **kwargs  # type: Any
     ):
-        """TestOne should be in an SecondVersionOperationsMixin. Returns ModelTwo.
+        """TestOne should be in an FirstVersionOperationsMixin.
```

Comment:
The changes in the test are expected and not necessarily a regression. Neither v1 or v2 are the default API. The way the dicts worked was we would get the description from the latest API version (unless the default API was not the latest API version). With the classes, we create a class object for each mixin operation, which results in the docstring coming from the first API version we see it appear in (unless the default API is different from the first API version).